tests/e2e/pkg/tests/mocks.go

```diff
@@ -0,0 +1,261 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+
+	//. "github.com/onsi/gomega"
```

Comment:
If we don't need this import it should be removed imo.

Reply:
fixed: [4031097](https://github.com/banzaicloud/koperator/pull/1020/commits/4031097bc70584321d1e3bf013d35a7fa92ad687)